web/js/containers/animation-widget.js

```diff
@@ -626,9 +627,11 @@ class AnimationWidget extends React.Component {
             onClose={onPushPause}
           />
         )}
-
-        {collapsed ? this.renderCollapsedWidget() : this.renderExpandedWidget()}
-
+        {!isDistractionFreeModeActive && (
```

Comment:
I'm kind of torn on whether or not to show the animation panel in distraction free mode.  It's kind of nice to be able to change or stop/start the animation while everything else is hidden.  But, by that logic you could just set it up and start/stop then enter distraction free mode.  🤷🏻‍♂️ 

I wonder if forcing the animation widget to show as collapsed during distraction free makes sense?

Reply:
I was torn on this, and eventually went with removing them both to stay with the "show more map" distraction free philosophy that we went with to recently nix the distraction free timeline.

Reply:
Yeah I totally support that 👍🏻